theme/management/commands/email_users.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management.base import BaseCommand
+from django.core.mail import send_mail
+from django.conf import settings
+from django.contrib.auth.models import User
+
+from theme.models import UserQuota, QuotaMessage
+
+
+class Command(BaseCommand):
+    help = "Send users emails for reporting over-quota usages and warnings, etc."
+
+    def add_arguments(self, parser):
+        parser.add_argument('--username', nargs='*')
+
+    def handle(self, *args, **options):
+        unames = options['username']
+        for uname in unames:
+            uemail = User.objects.filter(username=uname).first().email
+            uqs = UserQuota.objects.filter(user__username=uname).all()

```

Comment:
@pkdash Thanks for the info. But in this case, since I don't have a ```user``` instance, but only have a username as passed-in argument, directly filtering on UserQuota class is simpler.